Braintree.podspec

```
@@ -68,6 +68,7 @@ Pod::Spec.new do |s|
   s.subspec "ShopperInsights" do |s|
     s.source_files = "Sources/BraintreeShopperInsights/*.swift"
     s.dependency "Braintree/Core"
+    s.resource_bundle = { "ShopperInsights_PrivacyInfo" => "Sources/BraintreeShopperInsights/PrivacyInfo.xcprivacy" }
```

To match naming convention in other PRs
```suggestion
    s.resource_bundle = { "BraintreeShopperInsights_PrivacyInfo" => "Sources/BraintreeShopperInsights/PrivacyInfo.xcprivacy" }
```